package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Stylus",
-  "version": "1.5.12",
+  "version": "1.5.13",
```

Comment:
This looks accidental. I did it once myself too.

Reply:
I think @Mottie didn't use `npm version` command to update version number when tagging a release so the lock file is out of sync.